examples/classification/main.py

```diff
@@ -76,6 +88,16 @@ def main(argv):
     if config.dist_url == "env://":
         config.update_from_env()
 
+    mlflow.set_tracking_uri(osp.join(config.log_dir, 'mlruns'))
+    # ToDo: naming experiment problem
```

Comment:
what's the problem? on what line with which error message?

Reply:
for some reason it still creates experiment paths under natural number names(0, 1, 2) but not under the ones I passed to `mlflow.create_experiment() `

Reply:
ok, let's create experiments in default location. but the name should be specified for better navigation in the mlflow ui